pyproject.toml

```diff
@@ -23,10 +23,12 @@ requires = ["poetry_core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
```

Comment:
Is astroid not supported in version 3.7?

Reply:
python 3.8 was required by django 4.2 which was added to dev dependencies(for testing). I reduced the version of django to 3.2 and the python version to 3.7.2(minimal version required for astroid)